packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -88,13 +92,13 @@ MenuItem.propTypes = {
   to: PropTypes.string,
 };
 
-export const SimpleMenu = ({ label, children, id, as: asComp, ...props }) => {
+export const SimpleMenu = ({ label, children, id, as: asComp, size, ...props }) => {
   const menuButtonRef = useRef();
   const menuId = useId('simplemenu', id);
   const [visible, setVisible] = useState(false);
   const [focusedItemIndex, setFocusItem] = useState(0);
   const childrenArray = Children.toArray(children);
-  const Component = asComp || Button;
+  const Component = asComp || (size === 'S' ? StyledButtonSmall : Button);
```

Comment:
Maybe we can create a new constant here, to make it a bit more readable
```js
const defaultComponent = size === 'S' ? StyledButtonSmall : Button;
const Component = asComp || defaultComponent;
```

Reply:
Good idea. Implemented in 851b710fc230a62fe933e426ecb4cb9a89f7121d